compiler/luci/pass/src/QuantizationUtils.cpp

```diff
@@ -292,6 +292,14 @@ uint32_t cal_offset(loco::TensorShape &dimension, uint32_t *indices)
          indices[2] * dimension.dim(3).value() + indices[3];
 }
 
+uint32_t cal_offset_2d(loco::TensorShape &dimension, uint32_t *indices)
+{
+  assert(dimension.rank() == 2);
+  assert(sizeof(indices) / sizeof(*indices) == dimension.rank());
```

Comment:
what does `sizeof(indices)` give? and why?
what does `sizeof(*indices)` give? and why?

Reply:
My intention was to check the size of the indices, but I realized that I could simply fix the input argument to an array with a fixed size. I have made this change.